test/cypress/integration/29-host.dashboard.test.js

```diff
@@ -10,6 +10,7 @@ describe('host dashboard', () => {
 
     cy.get('[data-cy="host-apply-btn"]:not([disabled]):visible', { timeout: 30000 }).click();
     cy.get(`input[name="name"]`).type('Cavies United');
+    cy.get(`input[name="slug"]`).clear();
```

Comment:
Why?

Reply:
@Betree This test was broken when Cypress types the name, the slug is automatically filled and when it tries to type the slug, the result is "caviesunited" + slug.